src/record/mod.rs

```diff
@@ -17,6 +17,8 @@
 
 //! Contains record-based API for reading Parquet files.
 
-pub mod api;
 pub mod reader;
+mod api;
 mod triplet;
+
+pub use self::api::Row;
```

Comment:
Should we export `RowField` as well or will it work with just `Row`?

Reply:
I think we should not expose `RowField` (or `Field`). For primitive types, the getters will return Rust native types, such as `i32`, `bool` etc. We also need to define and expose clean-defined interfaces for array and map. The former should also have getters like `getBoolean(rowIndex: usize)` to return the value at a particular row, and the latter should have something like `getKeys()`, `getValues()`, etc. 

I'll take a further look into this. It seems rather verbose if we have to define `getXXX` for every data type on struct, array, etc. I wonder if there's a cleaner way to do it.